executable/optionrom.ksy

```diff
@@ -0,0 +1,474 @@
+meta:
+  id: optrom
```

Comment:
>To be frank, when it is just executable code, it usually means that Kaitai Struct has nothing to do with it.

It's true, "executable formats"  having no data tables for loaders are not executable formats.

>In case of option ROMs, as I see, .ksy describes tons of tables which are not executable code, but used by executable code to configure hardware stuff.

This is because the modules have not landed yet. In fact, it's better to move most of tables out of this file and load them as modules. In this case pci_data_struct,  pci_expansion_rom_header,  legacy_option_rom_header and pnp_hdr make ```executable``` file format and the most of other headers go to ```hardware```.  In fact some of the structs should go to ```firmware``` as they were meant to be used as containers for other headers (according to the spec), but ... I have not found any case they were really used in this way, so let them be a part of ```executable``` format.